independent-publisher-connectors/OK dokument/apiDefinition.swagger.json

```diff
@@ -157,7 +157,8 @@
                   "x-ms-visibility": "important",
                   "enum": [
                     "tosign",
-                    "preprocessing_tags"

```

Comment:
@SoliteaSlovensko An enum value has been removed from the input field. Existing flows may have this value chosen and will continue to send it. can you please confirm that this update will not break existing users' flow and apps?

Reply:
Removing this enum will not have impact on existing flows or apps, values defined in emuns are still valid and string send to request will work.